jupyterhub/spawner.py

```diff
@@ -212,6 +213,19 @@ def name(self):
             return self.orm_spawner.name
         return ''
 
+    # pass getattr to ORM spawner
+    def __getattr__(self, attr):
```

Comment:
Maybe @minrk can comment on why so far attributes were not automatically forwarded to the ORM class.

My Python intuition (potentially outdated) makes me feel a bit uncomfortable when I see classes implementing dunder methods for convenience. It seems like a good idea at the time but down the road it causes confusion/debugging pain. Usually the only advantage is the saving of a few characters when writing the original code.

Reply:
I copied this pattern from the `user.User` ORM wrapper and while it makes the client-side code for this object a bit cleaner (and less tightly coupled to the internals of this object) it does introduce some magic. I can see pros/cons both ways. I don't have an issue with removing this.